src/parser/ProcessControlSentences.cpp

```diff
@@ -0,0 +1,25 @@
+/* Copyright (c) 2019 vesoft inc. All rights reserved.
+ *
+ * This source code is licensed under Apache 2.0 License,
+ * attached with Common Clause Condition 1.0, found in the LICENSES directory.
+ */
+
+#include "parser/ProcessControlSentences.h"
+
+namespace nebula {
+std::string ReturnSentence::toString() const {
+    std::string buf;
+    buf.reserve(256);
+
+    buf += "RETURN $";
```

Comment:
if var_ is nullptr, here  has not ”RETURN $"

Reply:
Actually the var_ shall never be nullptr, or fatal error might occurred somewhere.